source/jormungandr/jormungandr/street_network/kraken.py

```diff
@@ -139,8 +139,21 @@ def _direct_path(
         response = instance.send_and_receive(req, request_id=request_id)
         if should_invert_journey:
             return self._reverse_journeys(response)
+
+        def has_bss_section(journey):
```

Comment:
Should this rather be like: 
```suggestion
        def has_bss_rent_before_put_back_section(journey):
```